xray/commands/offlineupdate/offlineupdate.go

```diff
@@ -154,7 +154,7 @@ func saveData(xrayTmpDir, filesPrefix, zipSuffix, targetPath string, urlsList []
 			DownloadPath:  url,
 			LocalPath:     dataDir,
 			LocalFileName: fileName}
-		_, err = client.DownloadFile(details, "", httputils.HttpClientDetails{}, 3, false)
+		_, err = client.DownloadFile(details, "", httputils.HttpClientDetails{}, false)
```

Comment:
I'm not convinced that there will be 3 attempts here.
If you agree with this, please change:
`httpclient.ClientBuilder().Build()` ->  `httpclient.ClientBuilder().SetRetries(3).Build()`

Reply:
Right. Done.